### PR TITLE
[v7r0] Fix Test_PoolComputingElement timings and number of processors evaluation

### PR DESCRIPTION
**Pull request by atsareg**
BEGINRELEASENOTES

*Resources
CHANGE: Test_PoolComputingElement - increase delays for job submission
CHANGE: PoolComputingElement - fix the logic of number of processors evaluation

ENDRELEASENOTES

### CODE REVIEW

File: Resources/Computing/PoolComputingElement.py [hunk 1]
```diff
@@ -181,11 +184,9 @@ def _getProcessorsForJobs(self, kwargs):
     # If there's a maximum number of processors allowed for the job, use that as maximum,
     # otherwise it will use all the remaining processors
     if 'maxNumberOfProcessors' in kwargs and kwargs['maxNumberOfProcessors']:
-      maxNumberOfProcessors = min(int(kwargs['maxNumberOfProcessors']), availableProcessors)
-    else:
-      maxNumberOfProcessors = availableProcessors
+      requestedProcessors = min(int(kwargs['maxNumberOfProcessors']), availableProcessors)
 
-    return maxNumberOfProcessors
+    return requestedProcessors
```
Comment (fstagni) [inline Resources/Computing/PoolComputingElement.py:189]: There might be a corner case when this is not set.

File: Resources/Computing/PoolComputingElement.py [hunk 2]
```diff
@@ -104,10 +104,13 @@ def submitJob(self, executableFile, proxy, **kwargs):
     self.pPool.processResults()
 
     processorsForJob = self._getProcessorsForJobs(kwargs)
+
+    print "****** processorsForJob", processorsForJob, executableFile
```
Comment (fstagni) [inline Resources/Computing/PoolComputingElement.py:108]: This was just debug, I guess.

File: Resources/Computing/PoolComputingElement.py (hunk 1 above)
Comment (atsareg) [reply Resources/Computing/PoolComputingElement.py:186]: Which is this case ?

File: Resources/Computing/PoolComputingElement.py (hunk 2 above)
Comment (atsareg) [reply Resources/Computing/PoolComputingElement.py]: Ah yes

**Review by fstagni — APPROVED** [review]

**Pull request merged**